opentelemetry-tracing/opentelemetry-java-tracing/spring-boot-kotlin/start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/zsh
+
+export OTEL_EXPORTER=zipkin
+export OTEL_ZIPKIN_ENDPOINT=http://localhost:8888
```

Comment:
This is untraditional port for SA and contradicts README

Reply:
Reverted to RM value.